data_steward/utils/bq.py

```diff
@@ -421,28 +421,6 @@ def update_labels_and_tags(dataset_id,
     return {**existing_labels_or_tags, **updates}
 
 
-def delete_dataset(project_id,
-                   dataset_id,
-                   delete_contents=True,
-                   not_found_ok=True):
```

Comment:
Thanks, this is good to remove as well! We can use the client directly. I checked for usages too and did not find any. I noticed list_datasets is redundant as well, but a different ticket can be created to remove it along with usages.

Reply:
👍 I created two different tickets yesterday.  They are in Daniel's Technical Debt epic.  See tickets 1778 and 1779.